stream/storage/impl/src/main/java/org/apache/bookkeeper/stream/storage/impl/metadata/RootRangeStoreImpl.java

```diff
@@ -80,7 +80,7 @@
     /**
      * ns name key: [NS_NAME_TAG][ns_name].
      */
-    static final byte[] getNamespaceNameKey(String nsName) {
+    static byte[] getNamespaceNameKey(String nsName) {
```

Comment:
Removing the "final" here just refactor since not needed for static methods?

Reply:
yes. cleans up some warnings in intellij.  it's a separate commit because it's logically separate.